core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -161,6 +161,17 @@
   public abstract T usePlaintext(boolean skipNegotiation);
 
   /**
+   * Makes the client use TLS.
+   *
+   * @return this
+   * @throws UnsupportedOperationException if transport security is not supported.
+   */
```

Comment:
also adding `* @since 1.9.0` in the doc would be nice

Reply:
Done. I went ahead and did that.